assets/components/introduction/circlesIntroduction.jsx

```diff
@@ -29,8 +29,8 @@ function CirclesIntroduction(props: PropTypes) {
       <SvgCirclesHeroDesktop />
       <SvgCirclesHeroMobileLandscape />
       <SvgCirclesHeroMobile />
-      <div className={`${classNameWithModifiers('component-circles-introduction__content', props.modifierClasses)} gu-content-margin`}>
-        <h1 className={classNameWithModifiers('component-circles-introduction__heading', props.modifierClasses)}>
+      <div className={`${classNameWithModifiers('component-circles-introduction__content ', props.modifierClasses)} gu-content-margin`}>
+        <h1 className={classNameWithModifiers('component-circles-introduction__heading component-circles-introduction__heading--compact', props.modifierClasses)}>
```

Comment:
`classNameWithModifiers` accepts a class name and a list of modifiers, in this case, we are passing another class name as a modifier which contradicts BEM style.